query-engine/connector-test-kit/src/test/scala/writes/withId/ExecuteRawSpec.scala

```diff
@@ -210,7 +210,8 @@ class ExecuteRawSpec extends WordSpecLike with Matchers with ApiSpecBase {
          |}
       """.stripMargin,
       project,
-      errorCode = errorCode,
+      errorCode = 2010,
+        // errorCode,
```

Comment:
we could add an argument to this assertion to check for the error metadata (it's in the JSON we return), that way we could check for the database error code here

Reply:
That would be really nice